spec/whois.md

```diff
@@ -0,0 +1,86 @@
+DID, the company can resolve the DID (included in the verifiable presentation
+from the student), and verify the [[ref: KEL]]. Then it can look in the DID's
+`whois` folder where it might find a couple of useful VCs or VPs with the
+`did:webs` DID as the subject. For example:
+
+- A verifiable presentation derived from a credential issued by [GLEIF]
+  containing the [LEI] of the school.
+  - Since the company knows about GLEIF, they can automate the lookup to get
+    more information about the school -- where it is registered, etc.
+- A verifiable presentation derived from a credential issued by the "Association
+  of Colleges and Universities" for the jurisdiction of the school.
+  - Since they don't know about the Association for that jurisdiction, the company can
+    repeat the process for the issuer of _that_ credential. They might (for
+    example), learn that the Association has in its `did:webs` `whois` folder a
+    verifiable credential issued by the government of the jurisdiction, and
+    the company recognizes that government.
+- A verifiable presentation derived from a credential issued by US News about
+  the school's placement on the [US News Rankings of Colleges and Universities].
+
+Such checks can all be done with a handful of HTTPS requests and the processing
+of the verifiable presentations. The result is an efficient, verifiable
+credential-based, decentralized trust registries.
+
+[GLEIF]: https://gleif.org
+[https://www.usnews.com/best-colleges/rankings/national-universities]: https://www.usnews.com/education/best-global-universities
+
+### `whois` folder Conventions
+
+The `whois` folder, if it exists, MUST be directly below the web root folder of
+the DID. The [DID URL path] to the folder MUST be `<did:webs DID>/whois`.
+
+The HTTPS path to the `whois` folder MUST be `<HTTPS URL conversion of did:webs
+identifier>/whois`. The Web Server that serves the `did:webs` documents MUST
+respond to a request for the HTTPS path to the `whois` folder with a list of the
+files within the folder so that an entity looking for information about the
+controller of the `did:webs` DID can see what verifiable credentials or
+verifiable presentations are available for review.
+
+The files in the `whois` folder must be [signed files](#signed-files), each with
+an associated [JSON Web Signature] file. The files in the folder must be either
+[verifiable credentials] or [verifiable presentations] with the verifiable
+credential subject `id` being the `did:webs` DID. If the file is a verifiable
+presentation, the verifiable presentation proof must be signed with the
+`did:webs` DID key(s). As with any [signed files](#signed-files), the
+accompanying [JSON Web Signature] file MUST be signed by the `did:webs` DID
+key(s).
+
+The verifiable credential or verifiable presentation format of the files can be
+(at least partially) identified by the file extension. The following extensions
```

Comment:
Shouldn't this be "recognized by IANA media type" rather than by file extension? The W3C has gone to a lot of trouble to talk about how credential types are recognized by media type...

Reply:
Definitely open to that — hence the “at least partially” weasel words 😄 .  I’m not the best on these standards...